packages/api/src/external/commonwell/link/shared.ts

```diff
@@ -44,6 +46,18 @@ export function patientWithCWData(
   return patientWithCW;
 }
 
+function isInsideOrgExcludeList(link: NetworkLink, orgIdExcludeList: Set<string>): boolean {
+  const identifiers = link.patient?.identifier || [];
+  return identifiers.some(id => {
+    const idSystem = id.system?.replace(/^urn:oid:/, "");
+    if (idSystem && orgIdExcludeList.has(idSystem)) {
```

Comment:
O(1)

Reply:
Also logs here so we can see if working in prod

Reply:
> O(1)

Actually O(n^2), since there's a loop inside a loop: `.some( () => { ... .has() } )`

Reply:
true true I was just talking about the lookup :)